src/ui/flutter_app/lib/shared/settings/settings_list_tile.dart

```diff
@@ -44,18 +43,16 @@ class SettingsListTile extends StatelessWidget {
     return ListTile(
       title: Text(
         titleString,
-        style: TextStyle(
-          fontSize: LocalDatabaseService.display.fontSize,
-          color: AppTheme.switchListTileTitleColor,
+        style: const TextStyle(
+          color: ColorSettings.switchListTileTitleColor,
         ),
       ),
       subtitle: subtitleString == null
           ? null
           : Text(
               subtitleString!,
-              style: TextStyle(
-                fontSize: LocalDatabaseService.display.fontSize,
-                color: AppTheme.listTileSubtitleColor,
+              style: const TextStyle(
+                color: ColorSettings.listTileSubtitleColor,
```

Comment:
Config is easier to associate with developers to avoid some of the parameterizable settings of Hard Code. The setting is more biased towards the setting used by users or operation and maintenance personnel. 

`listTileSubtitleColor` is not the color set by the user. Is it more accurate to change `Setting` to `Config`?

Reply:
I named the class that way as  we already had a Config and Colors class back then. 
We can have another look once the migration is finished as the name wasn't introduced with this pr